content/swiftui/concepts/views/terms/table/table.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: SwiftUI table
+Description: Table view in SwiftUI
+Subjects: Mobile Development, iOS application, SwiftUI
+Tags: SwiftUI, iOS, table, view
+
+---
+
+# SWIFTUI - LIST
+
+## **A container that presents rows of data arranged in one or more columns, optionally providing the ability to select one or more members**
```

Comment:
```suggestion
A **Table** is a container that presents rows of data arranged in one or more columns, optionally providing the ability to select one or more members.
```
This is not prper use for a header, and we only bold the first instance of the entry name.